DefensiveItem.cpp

```diff
@@ -3,10 +3,10 @@
 #include "Paladin.h"
 #include "DragonSlayer.h"
 #include "Dragon.h"
-#include <assert>
+#include "Utility.h"
+#include <cassert>
```

Comment:
**Don't include what you don't use**
Only include the header files that you are actually using
- [ ] revise